packages/newspringchurchapp/src/ui/ContentCardConnected/ContentCardConnected.js

```diff
@@ -24,14 +24,16 @@ const ContentCardConnected = memo(
               const coverImage = get(node, 'coverImage.sources', undefined);
               const hasMedia = !!get(node, 'videos.[0].sources[0]', null);
               const isLive = !!liveStream;
-              const labelText = get(node, 'parentChannel.name', '');
+              const labelText = get(node, 'parentChannel.name', '')
+                .split(' - ')
+                .pop();
```

Comment:
could proabably be `labelText = isLive ? "Live" : get...` and then just use `labelText` down below.

Reply:
although if you're matching what core is doing, leave it and we'll fix it there

Reply:
Yep, matching core here. Although I like your solution better. We should update core later.